spec/javascript/pages/browse/ListBrowser.spec.js

```diff
@@ -1,16 +1,24 @@
-import {assert} from 'chai'
-import {mount} from '@vue/test-utils'
+import {createLocalVue, shallowMount} from '@vue/test-utils'
+import {configure} from 'vue_config'
 import ListBrowser from 'pages/browse/ListBrowser'
-import testData from '../../../../lib/contributions.json'
+import testData  from '../../../../lib/contributions.json'
 
 describe('ListBrowser', () => {
-  it('works with reasonable data', function () {
-    // TODO - get this working again!!!
-    // const wrapper = mount(ListBrowser, {
-    //   propsData: {
-    //     contributions: testData.contributions,
-    //   },
-    // })
-    // assert.match(wrapper.text(), /look after my kid/i)
+  it('shows the respond column with typical data', function () {
+    const wrapper = shallowMount(ListBrowser, {
+      localVue: configure(createLocalVue()),
+      propsData: testData
+    })
+    var respondHeaders = wrapper.findAll('th').filter( header => header.text() == 'Respond')
+    assert.equal(1, respondHeaders.length)
+  })
+  it('hides the respond if there is no response urls', function() {
+    var responselessContributions = testData.contributions.map( contribution => { contribution.respond_path = ''; return contribution})
+    const wrapper = shallowMount(ListBrowser, {
+      localVue: configure(createLocalVue()),
+      propsData: {responselessContributions}
+    })
+    var respondHeaders = wrapper.findAll('th').filter( header => header.text() == 'Respond')
+    assert.equal(0, respondHeaders.length)
```

Comment:
Refactored a bit to showcase some rspec-like techniques you could consider:
```suggestion
  def('contributions', testData)

  def('wrapper', () => {
    return shallowMount(ListBrowser, {
      localVue: configure(createLocalVue()),
      propsData: $contributions,
    })
  })

  def('tableHeaders', () => $wrapper.findAll('th').wrappers)

  describe('respond column', () => {
    it('shows the column with typical data', function () {
      assert.isTrue($tableHeaders.some(header => header.text() == 'Respond'))
    })

    describe('when contributions do not come with response urls', () => {
      def('contributions', () => {
        testData.contributions.map(contribution => {
          contribution.respond_path = ''
          return contribution
        })
      })

      it('hides the respond button', function() {
        assert.isFalse($tableHeaders.some(header => header.text() == 'Respond'))
      })
```
Things to note:
* Use of `def` which is a like `let` in rspec. Helpful to extract common pieces (eg wrapper creation logic), thereby making the difference between examples more distinct.
* Use of `some` array method instead of `find` + `length`.